README.md

```diff
@@ -382,9 +375,9 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   @enabled = true unless defined?(@enabled)
   ~~~
 
-* Do not put a space between a method name and the opening parenthesis.
+* Avoid spaces between a method name and the opening parenthesis.
```

Comment:
I think this should also be kept as `do not`

Reply:
You might be right, as it can sometimes be a syntax error.

```ruby
def foo (a, b) # works
end

foo 1, 2       # works
foo(1, 2)      # works
foo (1, 2)     # SyntaxError: unexpected ',', expecting ')'
               # foo (1, 2)
               #       ^
```

Reply:
From my previous reply:

> It's not up to the style guide to make assumptions about "how strong this rule should be". I am using "avoid" as the consistent verb to advise developers about things they shouldn't do, and making exceptions depending on how strong we feel about a specific rule defeats the purpose of this very change.